.gitignore

```diff
@@ -263,4 +263,7 @@ __pycache__/
 *.pyc
 
 # cake
-tools/
\ No newline at end of file
+tools/
+/src/assets/FontAwesome5Brands.otf
```

Comment:
think you can revert this change

Reply:
Okay, but how do I have those files locally and have them ignored by git?

Reply:
why do you need these files there? The fonts are only supposed to be in MediaElement sample projects.

The `src/assets` folder is just for documentation assets, like images and branding. Not fonts for sample projects.

Reply:
You're right, we don't. It was a misunderstanding from my side. It's fixed now.